systemservices/marketplace/src/contracts/version.ts

```diff
@@ -30,6 +31,10 @@ const getServiceVersion = async (contract: Marketplace, versionHash: string): Pr
   }
   const version = await contract.methods.serviceVersion(versionHashHex).call()
   const manifestData = await getManifest(hexToAscii(version.manifestProtocol), hexToAscii(version.manifest))
+  if (!manifestData) {
+    console.warn('version', versionHash, 'is not a valid version')
```

Comment:
I think this console.warn is useless. There is already 2 console.warn in getManifest function. It should be enough

Reply:
removed